lib/configsetup.inc.php

```diff
@@ -1629,6 +1629,13 @@
             'name' => 'protect[localhost_index]',
             'value' => $config['protect']['localhost_index'],
         ],
+        'index_redirect' => [
+            'view' => 'advanced',
+            'type' => 'input',
+	        'placeholder' => $defaultConfig['protect']['index_redirect'],
```

Comment:
Space

Reply:
Hmm I don't see that in my ide. Maybe because I used tabs?

Reply:
```suggestion
	      'placeholder' => $defaultConfig['protect']['index_redirect'],
```